osmosis-1/osmosis.zone_assets.json

```diff
@@ -2972,6 +2972,26 @@
       },
       "osmosis_verified": false,
       "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "composable",
+      "base_denom": "ibc/3A0A392E610A8D477851ABFEA74F3D828F36C015AB8E93B0FBB7566A6D13C4D6",
```

Comment:
this base denom should be ibc/C58E5D2571042137CB68B1B9851C4E7211C05F7C2C79E21E0966AF0F063961F8
(The checks are failing because it's looking for ibc/3A... on composable, which doesn't exist because that's the denom on Osmosis)